docs/cloud/managing-permissions.md

```diff
@@ -0,0 +1,83 @@
+# Managing permissions
+
+This page covers managing permissions for Earthly Cloud products, such as Earthly Cloud Secrets, Earthly Satellites and Earthly CI.
+
+## Overview
+
+Earthly Cloud's permissions model has two security boundaries:
+
+* Earthly orgs
+* Earthly projects
+
+Users may be invited to a specific organization, and optionally to specific projects within the organization.
+
+Earthly orgs may contain the following shared resources:
+
+* Satellites
+* Earthly projects
+
+Earthly projects, in turn, may contain the following resources:
+
+* Secrets
+* Pipelines
+* Build history, including build logs
+
+## Earthly org access levels
+
+Within an Earthly org, users may be granted one of the following access levels:
+
+* `read`: Can view the org, including viewing satellites and projects.
```

Comment:
We could be verbose here, but it may also just ban expectation 🤷 
```suggestion
* `read`: Can view the org, including viewing satellites, projects, and user membership.
```